src/main.cc

```diff
@@ -178,6 +178,25 @@ signal_cb(ev_loop *loop, struct ev_signal *w, int revents)
 	tarantool_exit(0);
 }
 
+static void
+signal_int_cb(ev_loop *loop, struct ev_signal *w, int revents)
+{
+    (void) loop;
```

Comment:
Use tabs for indentation in C code, not spaces.

Reply:
I suggest https://github.com/vivien/vim-linux-coding-style if you're using Vim. It switched to tabs for C code even when spaces are set by default for all other file types.

Reply:
Fixed, I'm sorry for that.

Reply:
There are still some space/tab inconsistencies.

Reply:
Fixed.